process/physics.py

```diff
@@ -5164,7 +5164,7 @@ def outplas(self):
         )
         po.ovarre(
             self.outfile,
-            "Fraction of core radiation subtracted from P_L",
+            "Fraction reduction of radiation power along core profile",
```

Comment:
I think this is wrong: it's the fraction of the core radiation (e.g. 0.6) that we then subtract from the loss power, not the reduction in the radiation power (0.4).

Reply:
Sorry for the nit-picking.  As I see it, if I understand correctly:
- Fraction of radiation power along core profile = 0.6
- Fraction *reduction* of radiation power along core profile = 0.4